app/mailers/user_mailer.rb

```diff
@@ -81,4 +81,23 @@ def recent_activity_digest_email(user:)
       )
     end
   end
+
+  def notify_admins_funds_is_returned_to_group_account(admin:, bucket:, done_by:, archived_member:, amount:, group_account:)
```

Comment:
should rename to `notify_admins_funds_are_returned_to_group_account`

Reply:
Oh - thanks. Done.